src/map/vending.cpp

```diff
@@ -711,6 +711,20 @@ static int vending_autotrader_free(DBKey key, DBData *data, va_list ap) {
 }
 
 /**
+* Update vendor location
+* @param sd: Player's session data
+*/
+void vending_update_vendor_location(map_session_data &sd)
```

Comment:
```suggestion
void vending_update(map_session_data &sd)
```
After your last commit it is not only location that is getting updated.